src/components/BottomNavigation.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react-native/no-inline-styles */
+import * as React from 'react';
+import {Text, View} from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import FontistoIcons from 'react-native-vector-icons/Fontisto';
+
+const JavascriptComponent = () => {
```

Comment:
Eventually, these will all just be different files, right?

Reply:
correct